addons/apecloud-mysql/templates/_helpers.tpl

```diff
@@ -285,9 +278,28 @@ env:
   - name: KB_MYSQL_CLUSTER_UID
     value: $(KB_CLUSTER_UID)
   - name: KB_MYSQL_N
-    value: $(KB_REPLICA_COUNT)
+    value: $(KB_COMP_REPLICAS)
```

Comment:
built-in env KB_COMP_REPLICAS will be deleted in the future and please declare it in vars